app/models/application_form.rb

```diff
@@ -48,6 +48,10 @@ class ApplicationForm < ApplicationRecord
   BRITISH_OR_IRISH_NATIONALITIES = %w[GB IE].freeze
   MAXIMUM_NUMBER_OF_COURSE_CHOICES = 4
 
+  # Applications created after this date include a single personal statement
+  # instead of 2 personal statement sections
+  SINGLE_PERSONAL_STATEMENT_FROM = DateTime.new(2023, 2, 10, 9, 0)
```

Comment:
Should this be after deployment to prod?

Reply:
What happens if you switch the flag on, some applications are created (with a single statement), then you switch it off again. Presumably those applications still need to use the single statement flow? In other words you can't switch the feature off completely?

Reply:
@stevehook yeah, the date and time would have to be updated to some point after we deploy to prod (and turn the feature flag on).  And yeah, we can't really turn the feature flag off again very easily... 😬 🤞